public/data/kanishbodhwani.json

```
@@ -0,0 +1,62 @@
+{
+  "name": "Kanish Bodhwani",
+  "type": "personal",
+  "bio": "Open Source Enthusiast!",
+  "avatar": "https://github.com/kanishbodhwani.png",
+  "links": [
+    {
+      "name": "Personal portfolio",
+      "url": "https://kanishportfolio.netlify.app/",
+      "icon": "link"
+    },
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/kanishbodhwani",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Linkedin",
```

I recommend capitalizing the `i` here. What do you think?
```suggestion
      "name": "Follow me on LinkedIn",
```

---

Yeah LinkedIn is better. Changing it.